src/App.tsx

```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+import { Component } from 'react';
+import { Clock } from './components/Clock';
```

Comment:
```suggestion
import { Clock } from './components/Clock.tsx';
```
![image](https://user-images.githubusercontent.com/96409563/197152766-cd81b260-c0af-47e3-b4a1-3136de1eaa31.png)